man/create_hyperlink.Rd

```diff
@@ -93,6 +94,3 @@ write_formula(
 x = create_hyperlink(text = "test.png", file = "D:/somepath/somepicture.png")
 write_formula(wb, "Sheet1", startRow = 11, startCol = 1, x = x)
 }
-\seealso{
-\code{\link[=write_formula]{write_formula()}}
-}
```

Comment:
Removed link to `write_formula` in `seealso`, but added one in `description.

Reply:
Just FYI: I'm considering to make `write_` functions internal. They are somewhat artifacts from `openxlsx`. Users should use on `wb_add_` functions